lerobot/configs/robot/koch.yaml

```diff
@@ -37,3 +37,4 @@ cameras:
     fps: 30
     width: 640
     height: 480
+max_relative_target: [10.0, 10.0, 10.0, 10.0, 10.0, 15.0]
```

Comment:
I am sorry I was not specific enough. One value should be the default. What do you think?